task/net/src/main.rs

```diff
@@ -150,9 +150,17 @@ const RX_RING_SZ: usize = 4;
 /////////////////////////////////////////////////////////////////////////////
 // Main driver loop.
 
-/// Global count of passes through the driver loop, for inspection through a
-/// debugger.
-static ITER_COUNT: AtomicU32 = AtomicU32::new(0);
+#[derive(Count)]
+enum Event {
+    /// Global count of passes through the driver loop, for inspection through a
+    /// debugger.
+    Iter,
+    /// IP activity occurred on an iteration.
+    IpActivity,
+    /// Timer wakeups
+    TimerWake,
+}
+counters!(Event);
```

Comment:
These counters are turned into an enum so that we can generate the same structure that the `humility counters` command knows about. Alternatively, if we don't like going through the enum, we _could_ hand-write a struct that _looks like_ the ones generated by the `counters` crate, like
```rust

struct EventCounts {
    Iter: AtomicU32,
    IpActivity: AtomicU32,
    TimerWake: AtomicU32,
}

#[used]
static EVENT_COUNTERS: EventCounts = ...;
```
and `fetch_add` to them, instead.

Reply:
I don't mind the enum, personally.